simulator/src/ledger/dumper.rs

```diff
@@ -476,7 +493,7 @@ fn dump_resources<T: ReadableSubstateStore, O: std::io::Write>(
         );
 
         // DUMP non-fungibles
-        if !vault_info.resource_type.is_fungible() {
+        if !blueprint.blueprint_name.eq(FUNGIBLE_VAULT_BLUEPRINT) {
```

Comment:
I think this would be clearer if it was `blueprint.blueprint_name.eq(NON_FUNGIBLE_VAULT_BLUEPRINT)`

Reply:
good point